dora/core/server/worker/src/main/java/alluxio/worker/dora/DoraMetaManager.java

```diff
@@ -113,24 +109,7 @@ public void put(String path, FileStatus meta) {
       mMetastore.putDoraMeta(path, meta);
       return;
     }
-    if (mPopulateMetadataFingerprint) {
-      Fingerprint originalFingerprint =
-          Fingerprint.parse(status.get().getFileInfo().getUfsFingerprint());
-      if (originalFingerprint == null) {
-        originalFingerprint = Fingerprint.INVALID_FINGERPRINT;
-      }
-      Fingerprint newFingerprint =
-          Fingerprint.parse(meta.getFileInfo().getUfsFingerprint());
-      if (newFingerprint == null) {
-        newFingerprint = Fingerprint.INVALID_FINGERPRINT;
-      }
-      boolean contentMatched = originalFingerprint.isValid()
-          && newFingerprint.isValid()
-          && originalFingerprint.matchContent(newFingerprint);
-      if (!contentMatched) {
-        invalidateCachedFile(path, status.get().getFileInfo().getLength());
-      }
-    } else {
+    if (shouldInvalidatePageCache(status.get().getFileInfo(), meta.getFileInfo())) {
```

Comment:
Can you please rename `status` to `existingStatus` or `originalStatus` or something similar?
This will increase readability.